packages/apps/human-app/server/src/modules/statistics/statistics.service.ts

```diff
@@ -26,21 +26,22 @@ export class StatisticsService {
   async getOracleStats(
     command: OracleStatisticsCommand,
   ): Promise<OracleStatisticsResponse> {
-    const url = command.address;
+    const oracleCacheKey = command.address + command.token;
```

Comment:
I don't like how this is handled right now and it looks a lot like hack - we may discuss this tomorrow

Reply:
Issue: 1ed59b1b Reverted ("Add token authentication for Oracle statistics.")